code/modules/reagents/chemistry/reagents/blob.dm

```diff
@@ -118,7 +118,7 @@
 		volume = ..()
 		M.apply_damage(0.6*volume, BRUTE)
 
-
+*/
```

Comment:
if you're going to remove something, don't comment it out.

Reply:
Fixed.